### ISSUE TITLE
What would be the best strategy to handle objects of very different sizes?

### ISSUE THREAD

**Issue by gorinars** [TensorBox/TensorBox#155]
I am trying to use the tool for text detection. It works quite well with overfeat_rezoom.json (slightly worse with lstm and a little better with small tuning of the learning rate).

There seems to be a problem with too small / too long reference boxes, and also when many boxes are located close to each other. Small boxes seem to be filtered in annotation_to_h5, while long ones are  frequently not detected well.

I was wondering what could be (if any) a possible extension of the tool to improve the performance. I was trying to increase focus_size, which indeed degrades the accuracy. 
Intuitively it seems that a smaller grid size could help, but it makes impossible to use pre-trained model provided with the code.

Do you possibly have in mind some advice for this problem?

**Comment by ledor473** [comment]
:+1: I'm trying to detect objects of various size (depending on the distance of the object vs the camera).

**Comment by atique81** [comment]
It seems that this network is suitable for small objects. Do you guys have any idea about extending it to detect large objects , for example, detecting heads from profile-picture like images where the head occupies a large region of the image?
